setup.sh

```diff
@@ -0,0 +1,283 @@
+#!/bin/bash
+
+#
+# Doubtfire MacOS Development Install Script
+# =======================
+#
+# Provides commands to install Doubtfire
+#
+
+#
+# OS Detection functions
+#
+isMac() {
+    if [[ `uname` == Darwin ]]; then
+        # 0 = true
+        return 0
+    else 
+        return 1
+    fi
+}
+
+isLinux() {
+    if [[ `uname` == Linux ]]; then
```

Comment:
I think you may need to `==` against two strings here. It's usually a good idea to always compare against strings: `== "Linux"`.